### ISSUE TITLE
Test: Add E2E tests

### ISSUE THREAD

**Issue by sendilkumarn** [sendilkumarn/webpack-scaffold-pwa#23]

**Comment by dhruvdutt** [comment]
@rishabh3122 Can you pick this up?

**Comment by rishabh3112** [comment]
No, I wasn't able to setup docker on my system. Check [webpack/webpack-cli#87](https://github.com/webpack/webpack-cli/issues/87) if you have any solution.

**Comment by rishabh3112** [comment]
@dhruvdutt

**Comment by rishabh3112** [comment]
So if there is some other way to add e2e, I would like to contribute to this.
I also want `get-started` steps on this.
